<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchParkDialog</class>
 <widget class="QDialog" name="BatchParkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量添加停车场</string>
  </property>
  <widget class="QPushButton" name="ptnDelete">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>121</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnModify">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>121</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnAdd">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>121</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnSave">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnClose">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="parkTableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>620</width>
     <height>470</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>停车场编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>停车场名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>车位总数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>已停车位量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>自编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>备注</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>201</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>停车场编号*：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtParkNo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>由0-9的数字组成,固定长4位</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>车位总数量*：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtParkSpace"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>由0-9的数字组成</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>备        注：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtParkComments"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>停车场名称*：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtParkName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>191</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>停车场自编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtParkSelfNum"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>181</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>已停车位量 ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtUsedSpace"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
